library/src/main/java/com/getstream/sdk/chat/StreamChat.java

```diff
@@ -36,6 +43,7 @@
     public static LiveData<OnlineStatus> getOnlineStatus() {
         return onlineStatus;
     }
+    public static Markwon markwonInstance;
```

Comment:
Let's hide it behind interface. Exposed static field is hard to remove later. Also we might change the markdown library later, so it's also would be better to hide implementation. Could look like this:
```java
interface Markdown {
  void setText(TextView textView, String text)
}

class MarkdownImpl implements Markdown {

  private Markwon markwon = Markwon.builder(context)
                        .usePlugin(CorePlugin.create())
                        .usePlugin(LinkifyPlugin.create())
                        .usePlugin(ImagesPlugin.create())
                        .usePlugin(TablePlugin.create(context))
                        .usePlugin(StrikethroughPlugin.create())
                        .build();

  void setText(TextView textView, String text){
    markwon.setMarkdown(textView, message)
  }

 static Markdown instance() {
    // lazy initialisation
 }
}
```
And I don't really see a reason to initialise it in `StreamChat`. User might no even open channel, so there is no need to initialise markdown library at all. So I would move it back to `ChannelListItemViewHolder` and use about like this:
```java
MarkdownImpl.instance().setText(tv_last_message, StringUtility.getDeletedOrMentionedText(lastMessage));
```
And later we will inject such kind of instances.

Reply:
`ChannelListView` also has MarkDown parsing for last message as well as Channel.
So I set it on `StreamChat`